tests/integration/test_data/update_channel.nu

```diff
@@ -2,7 +2,7 @@
 export def main [channel: string] {
     let platforms = ["win-64", "linux-64", "osx-arm64", "osx-64"]
     cd $channel
-    rm -rf output
```

Comment:
Let's just add an exception for this one channel. In general this is pretty useful

Reply:
There currently isn't a case where this would do anything right?